modules/geoipbackend/geoipbackend.cc

```diff
@@ -547,7 +558,16 @@ string GeoIPBackend::format2str(string sformat, const Netmask& addr, GeoIPNetmas
     string rep;
     int nrep=3;
     tmp_gl.netmask = 0;
-    if (!sformat.compare(cur,3,"%cn")) {
+    if (!sformat.compare(cur,3,"%mp") && !s_mapping_lookup_formats.empty()) {
```

Comment:
Removed the check, it was to avoid checking the lookup formats if there is no mapping.

Now it will replace it by unknown.

I do not think we need to fail here or at loading time (or eventually to avoid mispelling in config).